features/promotion/managing_promotions/adding_promotion.feature

```diff
@@ -62,7 +62,7 @@ Feature: Adding a new promotion
         Given I want to create a new promotion
         When I specify its code as "FULL_METAL_PROMOTION"
         And I name it "Full metal promotion"
-        And I specify that it should starts at "21.04.2017"
-        And I specify that it should ends at "21.05.2017"
+        And I specify that it should start at "21.04.2017"
+        And I specify that it should end at "21.05.2017"
```

Comment:
Kinda don't like `I specify that it should` phrase.

I make it available from "21.04.2017" to "21.05.2017"?
I make it available since "21.04.2017"?
I make it available until "21.05.2017"?